docs/pages/blog/joining-vercel.mdx

```diff
@@ -3,7 +3,7 @@ title: Turborepo is joining Vercel
 date: 2021/12/09
 description: Turborepo is joining Vercel to make the web even faster.
 tag: web development
-ogImage: /og-image.png
+ogImage: /images/blog/joining-vercel/twitter-card.png
```

Comment:
Was `/og-image.png` a real thing or is this the address for the dynamically generated images?

Reply:
It still is a real thing in this PR. But it gets removed in the og-image generator PR.